src/Bindings/ManualBindings.cpp

```diff
@@ -266,6 +266,21 @@ static int tolua_StringSplit(lua_State * tolua_S)
 
 
 
+static int tolua_StringSplitWithQuotes(lua_State * tolua_S)
+{
+	cLuaState LuaState(tolua_S);
+	std::string str = (std::string)tolua_tocppstring(LuaState, 1, 0);
+	std::string delim = (std::string)tolua_tocppstring(LuaState, 2, 0);
```

Comment:
We have a nice template function `cLuaState::GetStackValues()` that is better at doing this - it can handle strings with embedded NULs etc. I'd prefer if you used it instead.